prisma/seed.ts

```diff
@@ -13,9 +13,11 @@ async function main() {
   await prisma.gender.createMany({
     data: [
       {
+        id: 1,
```

Comment:
Do we need to add the `id`s manually?

Reply:
Not really, I did it out of habit :). WIll fix!